time stamp
float32 yaw
float32 pitch